modules/util/osgi-bundle-builder/src/main/java/com/liferay/osgi/bundle/builder/commands/JarCommand.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.osgi.bundle.builder.commands;
+
+import aQute.bnd.osgi.Jar;
+
+import java.io.File;
+
+import java.nio.file.Files;
+
+/**
+ * @author David Truong
+ */
+public class JarCommand extends BaseCommand {
+
+	public void writeOutput(Jar jar) throws Exception {
+		File output = getOutput();
+
+		Files.createDirectories(output.toPath());
```

Comment:
What if `output` points to a file? AFAIK this method will create a directory with the file name.